src/attribute_generator.rs

```diff
@@ -55,7 +55,7 @@ impl OsGatewayAttributeGenerator {
     /// * The [Provenance Blockchain Account](https://docs.provenance.io/blockchain/basics/accounts)
     /// that signs the wasm payload must be the value owner of the
     /// [Provenance Blockchain Scope](https://docs.provenance.io/modules/metadata-module#metadata-scope)
-    /// OR the signer must be the same account as is used for [target_account_address](self::OsGatewayAttributeGenerator::access_revoke::target_account_address).
+    /// OR the signer must be the same account as is used for [target_account_address](self::OsGatewayAttributeGenerator::access_revoke).
```

Comment:
Minor cargo doc fix: you cannot link directly to parameters, so this link will just take the person clicking it to the function's documentation, where the value is readily available for maximum reading